desktop/core/src/desktop/lib/raz/raz_client_test.py

```diff
@@ -102,7 +102,8 @@ def setUp(self):
     self.raz_token = "mock_RAZ_token"
 
     self.s3_path = 'https://gethue-test.s3.amazonaws.com/gethue/data/customer.csv'
-    self.adls_path = 'https://gethuestorageaccount.dfs.core.windows.net/demo-gethue-container/demo-dir1/customer.csv'
+    self.adls_path = 'https://gethuestorage.dfs.core.windows.net/data/user/csso_hueuser?action=getStatus'
```

Comment:
This is not a path anymore, could we keep it clean? (so that we can reuse if for other actions)